### DISCUSSION TITLE
Real logins, cookies and DSGVO

### DISCUSSION THREAD

**Discussion by Long-HaHoang** [neuefische/questions#525] [Web]
Hi everybody,

the nextauth.js was really interesting and I just saw in the sprint, that some of you are using it.
Here is my question.
Don't need a cookie banner and a DSGVO page to inform the users about it.
Is our case an exception, since it's not a real website?
What's the best practice for a portfolio/capstone project?

**Comment by pixelass** [comment]
Wir sind aktuell noch in der Klärung zu diesem Thema.
